static/js/StaticPages.jsx

```diff
@@ -3126,6 +3132,330 @@ const JobsPage = memo(() => {
+                    נסו את המוצרים שמפתחי תוכנה וידידי ספריא מרחבי העולם בנו עבורכם! <a href="www.example.com">גלו את הפרויקטים</a>                
+                </HebrewText>  
+                <EnglishText>
+                Check out the products our software developer friends from around the world have been building for you! <a href="www.example.com">Explore</a> 
+                </EnglishText>
+            </InterfaceText>
+        </p>
+      </div>
+    );
+  };
+
+/**
+ * The following are the building block components of an individual product. 
+ */
+ 
+// The title and gray background label for each product
+const ProductTitle = ({product}) => {
+    return (
+        <div className='productsTitleAndLabel'>
+        <span className="productsTitle">
+            <InterfaceText text={{en: product.titles.en , he: product.titles.he }} />
+        </span>
+        {product.type.en ? (<span className="productsTypeLabel"> 
+            <InterfaceText text={{en: product.type.en , he: product.type.he }} />                               
+        </span>) : ''}
+    </div>
+    );
+};
+
+
+
+// The call-to-action (link) in the heading of each product
+// TODO - uncomment <OnInView /> once analytics is confirmed
+const ProductCTA = ({cta}) => {
+    return (
+
+        // <OnInView onVisible={() => productsAnalytics(product?.rank, `${product?.titles.en}_${cta.text.en}`, product?.type.en, "viewed")}>
+
+        // TODO - once analytics finalized, add onClick={productsAnalytics(product?.rank, `${product?.titles.en}_${cta.text.en}`, product?.type.en, "clicked")}
+        <a href={cta.url} >
+            {cta.icon.url && <img className="productsCTAIcon" 
+                                data-image-path={cta.icon.url} 
+                                src={cta.icon.url} 
+                                alt="Click icon" />}
+                                
+            <span className="productsCTA">
+                <InterfaceText text={{en: cta.text.en , he: cta.text.he }} />
+            </span>
+        </a>
+
+        // </OnInView>
+
+
+    );
+};
+
+// The main body of each product entry, containing an image and description
+const ProductDesc = ({product}) => {
+    return (
+        <div className="productsInner">
+            <img src={product.rectanglion.url} alt={`Image for product: ${product?.titles?.en}`}/>
+
+            <InterfaceText>  
+                <HebrewText>  
+                    <ReactMarkdown className="productsDesc">
+                        {product.desc?.he}
+                    </ReactMarkdown> 
+                </HebrewText>  
+                <EnglishText>
+                    <ReactMarkdown className="productsDesc">
+                        {product.desc?.en}
+                    </ReactMarkdown>
+                </EnglishText>
+            </InterfaceText>           
+        </div>
+    );
+};
+
+// The main product component, comprised of the building block sub-components
+const Product = ({id, product}) => {
```

Comment:
Why is the id being passed her if the key is assigned in the outer component while you map over the data creating "Products":
`ProductList.push(<Product key={product.id} product={product} />`

Reply:
You're right! Thanks for catching.